salt/version.py

```diff
@@ -773,38 +777,39 @@ def versions_report(include_salt_cloud=False):
 
 def msi_conformant_version():
     '''
-    An msi installer uninstalls/replaces a lower "internal version" of itself.
-    "internal version" is ivMAJOR.ivMINOR.ivBUILD with max values 255.255.65535.
-    Using the build nr allows continuous integration of the installer.
-    "Display version" is indipendent and free format: Year.Month.Bugfix as in Salt 2016.11.3.
-    Calculation of the internal version fields:
-        ivMAJOR = 'short year' (2 digits).
-        ivMINOR = 20*(month-1) + Bugfix
-            Combine Month and Bugfix to free ivBUILD for the build number
-            This limits Bugfix < 20.
-            The msi automatically replaces only 19 bugfixes of a month, one must uninstall manually.
-        ivBUILD = git commit count (noc)
-            noc for tags is 0, representing the final word, translates to the highest build number (65535).
+    For upgrades, an msi installer relies on the "internal version".
+    The "internal version" MAJOR.MINOR.BUILD is constrained to 255.255.65535.
+    This contraint can be satisfied for released versions but not so for nightly builds.
+    The nightly builds, therefore, must first be uninstalled.
 
-    Examples:
-      git checkout    Display version      Internal version    Remark
-      develop         2016.11.0-742        16.200.742          The develop branch has bugfix 0
-      2016.11         2016.11.2-78         16.202.78
-      2016.11         2016.11.9-88         16.209.88
-      2018.8          2018.3.2-1306        18.42.1306
-      v2016.11.0      2016.11.0            16.200.65535        Tags have noc 0
-      v2016.11.2      2016.11.2            16.202.65535
+    The "display version" is free format.
 
+    Examples:
+      Git-tag                msi display version         msi internal version
+      v3000                  3000                        30.00.0.0
+      v3001                  3001                        30.01.0.0
+      v3001.1                3000.1                      30.01.1.0
```

Comment:
msi display version needs to match the git-tag: `3001.1`

Reply:
Thanks. Corrected.

Reply:
@twangboy, I moved the mis internal version routine to salt-windows-msi PowerShell. 
I think this is the much better place.